helper/add.go

```diff
@@ -0,0 +1,5 @@
+package helper
```

Comment:
Not sure I understand the purpose of this?

Reply:
The `helper` package contains functions that we register for use in our templates. So at some point a dev would've added in this `add` helper function to make some of the presentation logic in the template less verbose or messy

We've not done anything with these helper functions other than port them over from `dp-frontend-renderer` as they are currently in use in our templates. So removing them would break those templates

I do think there is space to refactor some of the templates to not require, e.g., the `loop` helper function. However, I think that would likely come in a PR later on as we'd need to assess what templates are affected etc